### PR TITLE
Update installation-creating-azure-control-plane.adoc

### PR DESCRIPTION
**Pull request by nikhilprajapati-world**
As per doc bug raised https://issues.redhat.com/browse/OCPBUGS-29724, I prpopse to mention in the docs about the parameters to set the  VM size when using the templates,

Parameters to set the VM size when using the templates which using architecture as x86_64

$ az deployment group create -g ${RESOURCE_GROUP} \
  --template-file "<installation_directory>/05_masters.json" \
  --parameters masterIgnition="${MASTER_IGNITION}" \ 
  --parameters baseName="${INFRA_ID}" \
  --parameters masterVMSize="Standard_D4s_v3"

Parameters to set the VM size when using the templates which using architecture as ARM/ARCH64

$ az deployment group create -g ${RESOURCE_GROUP} \
  --template-file "<installation_directory>/05_masters.json" \
  --parameters masterIgnition="${MASTER_IGNITION}" \ 
  --parameters baseName="${INFRA_ID}" \
  --parameters masterVMSize="Standard_D4ps_v5"

Version(s):

Issue:

Link to docs preview:

https://docs.openshift.com/container-platform/4.15/installing/installing_azure/installing-azure-user-infra.html#installation-creating-azure-control-plane_installing-azure-user-infra

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/installation-creating-azure-control-plane.adoc [hunk 1]
```diff
@@ -63,11 +63,14 @@ ifdef::azure[]
 ----
 $ az deployment group create -g ${RESOURCE_GROUP} \
   --template-file "<installation_directory>/05_masters.json" \
-  --parameters masterIgnition="${MASTER_IGNITION}" \ <1>
-  --parameters baseName="${INFRA_ID}" <2>
+  --parameters masterIgnition="${MASTER_IGNITION}" \ <1>
```
Comment (mjpytlak) [inline modules/installation-creating-azure-control-plane.adoc:66]:
```suggestion
  --parameters masterIgnition="${MASTER_IGNITION}" \// <1> 
```

File: modules/installation-creating-azure-control-plane.adoc [hunk 2]
```diff
@@ -63,11 +63,14 @@ ifdef::azure[]
 ----
 $ az deployment group create -g ${RESOURCE_GROUP} \
   --template-file "<installation_directory>/05_masters.json" \
-  --parameters masterIgnition="${MASTER_IGNITION}" \ <1>
-  --parameters baseName="${INFRA_ID}" <2>
+  --parameters masterIgnition="${MASTER_IGNITION}" \ <1> 
+  --parameters baseName="${INFRA_ID}" \ <2>
```
Comment (mjpytlak) [inline modules/installation-creating-azure-control-plane.adoc:67]:
```suggestion
  --parameters baseName="${INFRA_ID}" \// <2>
```

File: modules/installation-creating-azure-control-plane.adoc [hunk 3]
```diff
@@ -63,11 +63,14 @@ ifdef::azure[]
...
+  --parameters masterVMSize="MASTER_VM_SIZE" <3>
 ----
 <1> The Ignition content for the control plane nodes.
 <2> The base name to be used in resource names; this is usually the cluster's infrastructure ID.
+<3> If the architecture is x86_64 then vm type should be "Standard_D4s_v3", or if the architecture is ARM/ARCH64 then vm  type must be "Standard_D4ps_v5"
```
Comment (mjpytlak) [inline modules/installation-creating-azure-control-plane.adoc:72]:
```suggestion
<3> If the architecture is x86_64, then the vm type should be `Standard_D4s_v3`. If the architecture is ARM/ARCH64, then the vm  type must be `Standard_D4ps_v5`.
```

**Review by mjpytlak — COMMENTED** [review]
@nikhilprajapati-world Some feedback based on a peer review.

**Comment by mjpytlak** [comment]
@jinyunma PTAL for QE. Thank you.

**Comment by mjpytlak** [comment]
@kelbrown20 I see you are assigned to https://issues.redhat.com/browse/OCPBUGS-29724. FYI.

**Comment by kelbrown20** [comment]
Updates made in https://github.com/openshift/openshift-docs/pull/72310

**Comment by mjpytlak** [comment]
Closing at @kelbrown20 request.

**Pull request closed without merging**